pixel/patterns/footer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Title: Default footer
+ * Slug: pixel/footer
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"layout":{"inherit":"true","type":"constrained"},"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","right":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60"}}}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)"><!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size">
+    <?php
+			printf(
+				/* Translators: WordPress link. */
+				esc_html__( 'Pixel Theme, proudly powered by %s', 'archeo' ),
```

Comment:
set font weight of the text `Pixel theme` to `500`.
and update theme name to `pixel`

Reply:
There is not a straightforward way to change the `appearance` of the text in line without making the block markup overly complex, so I just made Pixel Theme a bold element.